app/controllers/ems_common.rb

```diff
@@ -185,6 +185,18 @@ def show
                       pluralize(@view.extras[:total_count] - @view.extras[:auth_count], "other #{title.singularize}") +
                       " on this " + ui_lookup(:tables => @table_name)
       end
+    elsif @display == "container_builds" || session[:display] == "container_builds" && params[:display].nil?
+      title = ui_lookup(:tables => "container_builds")
+      drop_breadcrumb(:name => @ems.name + " (All #{title})",
+                      :url  => show_link(@ems, :display => @display))
+      @view, @pages = get_view(ContainerBuild, :parent => @ems)  # Get the records (into a view) and the paginator
+      @showtype = @display
+      if @view.extras[:total_count] > @view.extras[:auth_count] && @view.extras[:total_count] &&
```

Comment:
Can `@view.extras[:total_count]` be `nil` or `false`? If yes, then this `if` will raise _undefined method `>' for nil:NilClass_. If no, then `&& @view.extras[:total_count] &&` part is not needed.